### PR TITLE
Update README & Other documentation

### PR DESCRIPTION
**Pull request by harry1453**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,115 @@
...
-will take the old `nxt-default.properties`/`nxt.properties` files and
-create `brs-default.properties.converted`/`brs.properties.converted`
-files in the conf directory. This should give you a headstart with the
-new option naming system.
 
-#### Windows
+The important part is that the Java version starts with `1.8` (Java 8)
 
-###### MariaDb
+If you do not have Java 8 installed, download it from [Oracle's Website](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html)
```
Comment (ohager) [inline README.md]: not openjdk?

File: README.md [hunk 2]
```diff
@@ -1,121 +1,115 @@
...
+##### Debian
 
+A `.deb` package is available [here](https://github.com/burst-apps-team/burstcoin-packages/releases/tag/v2.3.0).
 
-**Note (H2 only):**  
-H2 dump username and password has been changed through updates. The current dump has the user 'sa' and password 'sa' which is used in the BRS 2.2.1 image. If you get username errors, please mount your custom config for your H2 file.
+##### Docker
 
-Old username and passwords:
+[Docker repository](https://hub.docker.com/r/burstappsteam/burstcoin)
```
Comment (ohager) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -137,59 +132,99 @@ services:
...
-If it's not, add it: 
+Once you have done this, look through the existing properties if there is anything you want to change.
+
+### Testnet
+
+Please see the [Wiki article](https://burstwiki.org/en/testnet/) for details on how to setup a testnet node.
+
+### Private Chains
+
+In order to run a private chain, you need the following properties:
```
Comment (ohager) [inline README.md]: valuable chapter

File: doc/Credits.md [hunk 4]
```diff
@@ -1,70 +1,83 @@
...
-
-@4nt1g0
-* helping with JOOQ migration
+# Contributors
+
+Numerous people have contributed to make this software what it is today. The Nxt-developers before Burst was forked off from the Nxt code base, the initial - yet anonymous - [Creator of Burstcoin](https://github.com/BurstProject) who invented PoC and also many small contributors who helped out with small fixes, typos, translations etc. In case we forgot to mention someone, please do not hesitate to bring this to our attention.
+
+## [Burst Apps Team](https://github.com/burst-apps-team)
+
+[Harry1453](https://github.com/harry1453) (Donation Address: [BURST-W5YR-ZZQC-KUBJ-G78KB](https://explore.burstcoin.network/?action=account&account=16484518239061020631))
```
Comment (ohager) [inline doc/Credits.md]: this is our man (pride) :grinning:

File: doc/Credits.md [hunk 5]
```diff
@@ -1,70 +1,83 @@
...
+* Introduced gRPC API
+* Heavy optimization of sync speed, along with lots of other database optimization work
+* Reverse lookup of Multi-out transactions
+* Auto pop-off feature
+* Lots of internal migration eg. json-simple -> gson, new versioning system
+* Lots of bug fixes
+* Lightweight basic desktop GUI
 
-@ac0v
+## [Proof of Capacity Consortium](https://github.com/poc-consortium) (Previous Developers)
```
Comment (ohager) [inline doc/Credits.md]: Respect to whom merits....nice one!

File: doc/Credits.md (hunk 4 above)
Comment (blankey1337) [reply doc/Credits.md]: 🥂

Comment (harry1453) [reply doc/Credits.md]: Thanks!

**Pull request merged**